intyg-proxy-service/integration-fakehsa/src/main/java/se/inera/intyg/intygproxyservice/integration/fakehsa/repository/FakeHsaRepository.java

```diff
@@ -67,6 +70,16 @@ public HealthCareUnit getHealthCareUnit(String id) {
     return healthCareUnitConverter.convert(parsedCareUnit);
   }
 
+  public List<HealthCareProvider> getHealthCareProvider(String id) {
+    final var parsedProvider = careProviderMap.get(id);
```

Comment:
Check if we are supposed to use this map. This only has name and id so HealthCareProvider object will be rather empty